src/main/java/model/data/DataTableConversionBuilder.java

```diff
@@ -34,6 +36,16 @@ public void addRowsFromTable(DataTable table) {
 			createRow(row);
 		}
 	}
+	
+	/**
+	 * Add all rows from a List to the builder.
+	 * @param table list of datarows.
+	 */
+	public void addRowsFromList(List<DataRow> table) {
+		for (Object obj : table) {
```

Comment:
Omdat je nu een generic type hebt, hoef je niet meer Object te doen, maar kan je gelijk `DataRow row : table` doen, dan hoef je ook niet meer te casten. Oh en ik denk dat rowList wel een betere naam is dan table.